plugins/reach/plugins/OpenCalais/batch/Constants.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+
+abstract class Constants
```

Comment:
rename file / class name as Constants is too common name + already have such file name in symphony,
Lets add some of the plugin name as prefix.

Reply:
@TalDubovKaltura could you please take care of this comment?

Reply:
@hilak Done